src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -430,6 +430,10 @@
                 <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
                 <target>Die Dateiendung ist ungültig ({{ extension }}). Gültige Dateiendungen sind {{ extensions }}.</target>
             </trans-unit>
+            <trans-unit id="111">
+                <source>The detected encoding "{{ detected }}" does not match one of the accepted encoding: "{{ encodings }}".</source>
+                <target>Die erkannte Kodierung "{{ detected }}" ist nicht gültig. Gültige Kodierungen sind "{{ encodings }}".</target>
```

Comment:
To be more precise:

```suggestion
                <target>Die erkannte Zeichenkodierung "{{ detected }}" ist nicht gültig. Gültige Kodierungen sind "{{ encodings }}".</target>
```

Alternatively "Zeichensatz" instead of "Zeichenkodierung".

Reply:
I was wondering indeed if „Zeichensatz“ wouldn’t be better.